monkey/infection_monkey/ransomware/ransomware_readme.txt

```diff
@@ -1,2 +1,49 @@
+ ░███   ░░███  ██████  ████████  ░░░  ███████      ░███    ░███  ██████   ████████   ████   ██████
+ ░███    ░███ ███░░███░░███░░███     ░░░███░       ░██████████  ░░░░░███ ░░███░░███ ░░███  ███░░███
+ ░███    ░███░███ ░███ ░███ ░███       ░███        ░███░░░░░░    ███████  ░███ ░███  ░███ ░███ ░░░
+ ░███    ███ ░███ ░███ ░███ ░███       ░███ ███    ░███         ███░░███  ░███ ░███  ░███ ░███  ███
+ ██████████  ░░██████  ████ █████      ░░█████     █████       ░░████████ ████ █████ █████░░██████
+░░░░░░░░░░    ░░░░░░  ░░░░ ░░░░░        ░░░░░     ░░░░░         ░░░░░░░░ ░░░░ ░░░░░ ░░░░░  ░░░░░░
+
+                                               -yddy-
+                                   ``          yN::Ny          ``
+                                 `ymdmo        .hNNh.  .     omdmy`
+                                 :Ny:dm.      .`-NN-  -m    .md:yN:
+                                  :sdNN:     :m:+NN+:ymy -` :NNds:
+                                     +Nmo-/ohNNmNNNNNNNdms-+mN+
+                         ```          dNNNNNNNNNNNNNNNNNNNNNNd          ```
+                       `ymhms      `+dNNNNNNNNNNNNNNNNNNNNNNNNd+`      smhmy`
+                       :Ny:dNh-   +mNNNNNNNNNNNNNNNNNNNNNNNNNNNNm+   -hNd:yN:
+                        -ossydNmhmNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNmhmNdysso-
+                              .dNNNNNNNNmhyyhdNNNNNNNNdhyyhmNNNNNNNNh.
+                          ./ooodNNNNNms-  ..` `/hNNh/` `..  -omNNNNNdooo/.
+                        /dNNNNNNNNNNd. :hNNNNms. ++ .smNNNmy: .dNNNNNNNNNNd/
+                       yNNmo:-mNNNNm. +NNmyodNNm.  .mNNdoymNN+ .mNNNNm-:omNNy
+                      /NNN-  -NNNNNm  hNNy  .NNN/  /NNN.  yNNh  mNNNNN-  -NNN/
+                      /NNN.  :NNNNNN: :mNNdhmNNh`  `hNNmhdNNm: :NNNNNN:  .NNN/
+                      `dNNd/..NNNNNNm/ .ohmmdy/ ```` /ydmmho. /mNNNNNN.`/dNNd`
+                       `omNNNmNNNNNNNNh+-`     :h::h:     `-+hNNNNNNNNmNNNmo`
+                         `:oyyymNNNNNNNNNmh+`          `+hmNNNNNNNNNmyyyo:`
+                              `hNNNNNNNNNs`/yys+/::/+syy/`yNNNNNNNNNh`
+                        ./+++ymNmNNNNNNNNd    `-://:-`    dNNNNNNNNmNmy+++/.
+                       -mh+dNd/` `sNNNNNNNo              oNNNNNNNs` `/dNd+hm:
+                       .ddsmh`     -ymNNNNNh-          -hNNNNNms-     `hmsdd.
+                         --.         `dNNNNNNds/:--:/sdNNNNNNd`         .--
+                                       :NNs/oydmNNNNNNNNmdyo/sNm:
+                                  .+hNN/      `.sNNs.`      /NNy+.
+                                 :my/dm.        -NN-        .md/ym-
+                                 .ddymy        `yNNy`       `ymydd.
+                                   .-.         yN//Ny         .-.
+                                               :dmmd:
+
+Infection Monkey is an open-source breach and attack simulation (BAS) platform. The files in this
+directory have been manipulated as part of a ransomware simulation. This is NOT a ransomware attack.
```

Comment:
This kind of is a ransomware attack or at least we're trying to get as close as possible to one. Maybe rephrase to `This is NOT a real ransomware attack.`?